docs/Release_notes.md

```diff
@@ -1,6 +1,59 @@
 V2X-Hub Release Notes
 ---------------------------------
 
+Version 7.7.0, released Aug 14th, 2023
+--------------------------------------------------------
+
+**Summary:**
+V2X Hub release 7.7.0 introduces key enhancements and fixes to improve data streaming, upgrade MySQL to version 8.0, and support multi-architecture Docker deployments. This release includes updates to initialization scripts and addresses clock reference issues in simulation mode. Additionally, this release removes obsolete Docker tags and makes the SENSOR_JSON_FILE_PATH an optional environment variable to enhance flexibility in simulation setups.
+
+**<ins>V2X Hub CDASim Functionalities </ins>**
+
+Enhancement in this release:  
+- V2X-Hub PR 611: Updated TCR message oldest field to use data sent by carma-platform in simulation mode to address clock reference issues.
+
+Fixes in this release: 
+- V2X-Hub PR 616: Updated SENSOR_JSON_FILE_PATH as optional environment variable in simulation since spawning sensors is not required for the base line functionality of V2X-Hub. Before this update CDASimAdapter would accept empty SENSOR_JSON_FILE_PATH files but not missing ones.
+
+Other Enhancements: 
+- V2X-Hub PR 612: Updated MySQL to 8.0 to support multi-architecture Docker deployment, removed architecture-based configuration folders, and updated initialization scripts.
+- V2X-Hub PR 615: Updated the initialization script to generate a .env file based on user input.
+  
+Other Fixes:
+- V2X-Hub PR 614: Removed the "version:" tag at the top of the docker-compose files. These are no longer needed and are obsolete in current and future versions of Docker.
+
+
+Version 7.6.0, released April 16th, 2024
```

Comment:
Based on the Platform release notes and on SharePoint, this should be April 10th, not 16th? 

I see that 7.6.0 release notes were missed somehow for Lavida. So good with this as long as it matches what we sent the client for approval.

Reply:
I've added 7.6.0 notes from client approval document in Share Point.